presto-verifier/src/main/java/com/facebook/presto/verifier/framework/JdbcPrestoAction.java

```diff
@@ -182,7 +182,7 @@ private PrestoConnection getConnection(
     {
         PrestoConnection connection = DriverManager.getConnection(
                 clusterUrls.get(queryOrigin.getCluster()),
-                configuration.getUsername(),
+                configuration.getUsername().orElse(null),
```

Comment:
What's the motivation to allow absent username?

Reply:
``java.sql.DriverManager`` treats `user` and `password` equally, as nullable parameters, so we don't need to arbitrarily put a restriction on `user`.